### PR TITLE
fix: add image thumbnail back to the feedback card

### PR DESCRIPTION
**Pull request by DragosIorgulescu**
## Changes

- added the image component back to the feedback card (was removed as a part of the footer component)

## Preview

Screenshot 2024-01-17 at 10 11 17

Screenshot 2024-01-17 at 10 14 41
## Events

n / a

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

MI-16 #done

### CODE REVIEW

File: packages/shared/src/components/cards/v1/FeedbackCard.tsx [hunk 1]
```diff
@@ -64,14 +70,29 @@ export const FeedbackCard = ({
...
+              {post.summary}
+            </p>
+          )}
+        </div>
+
+        <ImageComponent
+          alt="Post Cover image"
+          src={post.image}
+          fallbackSrc={cloudinary.post.imageCoverPlaceholder}
+          className={classNames('object-cover')}
```
Comment (rebelchris) [inline packages/shared/src/components/cards/v1/FeedbackCard.tsx:88]: You don't need `classNames` here

**Review by rebelchris — APPROVED** [review]
One small remark, non blocking.

File: packages/shared/src/components/cards/v1/FeedbackCard.tsx (hunk 1 above)
Comment (DragosIorgulescu) [reply packages/shared/src/components/cards/v1/FeedbackCard.tsx]: good catch 🙏

**Review by kopancek — APPROVED** [review]
LGTM 👍

**Review by nensidosari — APPROVED** [review]
Looks good!

File: packages/shared/src/components/cards/v1/FeedbackCard.tsx [hunk 2]
```diff
@@ -64,14 +69,29 @@ export const FeedbackCard = ({
...
-            {post.summary}
-          </p>
-        )}
-      </div>
+
+      <CardContent className="rounded-14 border border-theme-divider-tertiary p-4">
+        <div className="mr-2 flex-1">
+          <h2 className="mb-2 line-clamp-1 typo-body">{post.title}</h2>
+          {post.summary && (
+            <p className=" line-clamp-2 text-theme-label-quaternary typo-callout">
```
Comment (subashcs) [inline packages/shared/src/components/cards/v1/FeedbackCard.tsx:77]: Formatting issue: Extra Space

**Pull request merged**